app/assets/stylesheets/custom_devise.scss

```diff
@@ -0,0 +1,44 @@
+.devise-background {
+  background-image: url("https://images.unsplash.com/photo-1429305336325-b84ace7eba3b?q=80&fm=jpg&s=8cf84b970497806cc96a06d6aba2b86b");
+  background-position: center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  height: 100%;
+  padding: 45px 20px 20px 20px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `45px 20px 20px` instead of `45px 20px 20px 20px`